demos/projects/ST/b-l475e-iot01a/port/azure_iot_flash_platform_port.h

```diff
@@ -13,7 +13,7 @@
 
 typedef struct AzureADUImageContext
 {
-    void * xUpdatePartition;                   /**< TODO: Fill in for ST partition. */
+    uint8_t * xUpdatePartition;               /**< Partition address for ST */
     uint8_t * pucBufferToWrite;               /**< The buffer containing the bytes to write to the flash. */
     uint32_t ulBytesToWriteLength;            /**< The length of the buffer from which to write the bytes. */
```

Comment:
@danewalton do we need these two (pucBufferToWrite and ulBytesToWriteLength)? They don't seem to be used anywhere when I do a Find All References in this project or the esp32 project.

Reply:
No if the fields aren't used then feel free to delete them.